File: U5/Java/U5P_8.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompan.u5p_8;
import java.util.Random;
import java.util.Scanner;
/**
 *
 * @author alfre
 */
public class U5P_8 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("¿Cuántas veces deseas simular el lanzamiento de dados? ");
        int vecesASimular = scanner.nextInt();

        for (int i = 1; i <= vecesASimular; i++) {
            System.out.println("\nSimulación " + i + ":");
            lanzarDados();
        }
    }

   
    private static void lanzarDados() {
        Random random = new Random();

       
        int dado1 = random.nextInt(6) + 1;
        int dado2 = random.nextInt(6) + 1;

        // resultados 
        System.out.println("Dado 1: " + dado1);
        System.out.println("Dado 2: " + dado2);
        System.out.println("Total: " + (dado1 + dado2));
    }
}

File: U5/C#/U5P_8.cs
using System;

namespace U5P_8
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Write("¿Cuántas veces deseas simular el lanzamiento de dados? ");
            int vecesASimular = Convert.ToInt32(Console.ReadLine());

            for (int i = 1; i <= vecesASimular; i++)
            {
                Console.WriteLine($"\nSimulación {i}:");
                LanzarDados();
            }
        }

        private static void LanzarDados()
        {
            Random random = new Random();

            int dado1 = random.Next(1, 7);
            int dado2 = random.Next(1, 7);

            // resultados 
            Console.WriteLine($"Dado 1: {dado1}");
            Console.WriteLine($"Dado 2: {dado2}");
            Console.WriteLine($"Total: {dado1 + dado2}");
        }
    }
}
